assets/app/scripts/controllers/build.js

```diff
@@ -69,6 +69,7 @@ angular.module('openshiftConsole')
                 };
               }
               $scope.build = build;
+              $scope.logCanRun = !(_.includes(['New', 'Pending'], build.status.phase));
```

Comment:
Shall I handle that [when I move on to fix this issue](https://github.com/openshift/origin/issues/5913)?